### DISCUSSION TITLE
好像是新版专栏（实为新版动态）

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#3986] [综合讨论 / General]
URL 格式为 `https://www.bilibili.com/opus/******************`，结尾那串十八位的数字和投稿了该文章的动态是一致的
例子：https://www.bilibili.com/opus/756507714292547617
![image](https://user-images.githubusercontent.com/49396421/218079312-0521cead-bc36-4ee6-98ae-394a008db173.png)
对应投稿文章动态：https://t.bilibili.com/756507714292547617

（夜间模式居然好像不用再怎么重新适配）

**Comment by Tinhone** [comment]
https://www.bilibili.com/opus/761009926636044305
https://t.bilibili.com/761009926636044305

普通动态也可以

**Comment by the1812** [comment]
> 例子：https://www.bilibili.com/opus/756507714292547617

我打开后被重定向到 https://www.bilibili.com/read/cv21473109 了

看图片应该就是新版动态的界面, 最早在 #3917 发现

**Reply by Geeyun-JY3** [reply]
那看来应该是专栏在 URL 上也被统一进动态不作区分了。

**Reply by Geeyun-JY3** [reply]
挺怪的，找了个长一点的专栏文章改 URL 又被重定向到 `https://www.bilibili.com/read/cv********` 了
